app/views/foods/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Foods</title>
+  <%= stylesheet_link_tag 'application', 'foods_index' %>
+</head>
+<body>
+
+<div class="btn-div">
+  <%= link_to "Add Food", new_food_path, class: "button" %>
+</div>
+
+<table>
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Measurement Unit</th>
+      <th>Unit Price</th>
+      <th>Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @foods.each do |food| %>
```

Comment:
- [ ] Please, in accordance to the project requirement the food details should only be accessible and seen by the logged-in user. I sign-up with another account and was able to see the food details created for previous account. Kindly correct this.

<img width="612" alt="Untitled" src="https://github.com/lerfast/recipe-app/assets/103386183/03de1e83-745c-4864-b9cc-bf127bc16a9c">

Reply:
I modify the code so only display the list of the food added by the logged-in user
![image](https://github.com/lerfast/recipe-app/assets/111823606/c709eeba-d506-40e7-ba0d-d2f17b46bbb6)

Reply:
- Well resolved